fenix/app/src/main/java/org/mozilla/fenix/debugsettings/ui/DebugDrawerHome.kt

```diff
@@ -24,24 +24,32 @@ import androidx.compose.runtime.rememberCoroutineScope
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
 import kotlinx.coroutines.launch
 import mozilla.components.support.ktx.android.content.appName
 import mozilla.components.support.ktx.android.content.appVersionName
+import org.mozilla.fenix.R
 import org.mozilla.fenix.compose.Divider
 import org.mozilla.fenix.compose.annotation.LightDarkPreview
 import org.mozilla.fenix.compose.inComposePreview
 import org.mozilla.fenix.compose.list.TextListItem
+import org.mozilla.fenix.debugsettings.navigation.DebugDrawerDestination
 import org.mozilla.fenix.theme.FirefoxTheme
 
+/**
+ * The navigation route for [DebugDrawerHome].
+ */
+const val DEBUG_DRAWER_HOME_ROUTE = "debug_drawer_home"
+
```

Comment:
Since the `Home` enum is no longer necessary, the route string now lives with its Composable.